apps/community/src/app/member/professor/page.css.ts

```diff
@@ -0,0 +1,20 @@
+import { style } from '@vanilla-extract/css';
+
+export const professorListWrapper = style({
+  display: 'grid',
+  gap: 16,
+  gridTemplateColumns: '1fr',
+  marginTop: 42,
```

Comment:
```suggestion
  gap: '1rem',
```

후추 리펙토링, DS 도입을 용이하기 위해 단위를 `rem` 으로 지정해주세요